config/dev.webpack.config.js

```diff
@@ -4,7 +4,19 @@ const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPl
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
     rootFolder: resolve(__dirname, '../'),
-    debug: true
+    debug: true,
+    useProxy: process.env.USEPROXY ? true : false,
+    localChrome: process.env.INSIGHTS_CHROME,
+    customProxy: process.env.USEPROXY ? [
```

Comment:
Shouldn't this be `API_ENDOINT`?
```suggestion
    customProxy: process.env.API_ENDOINT ? [
```

Reply:
nope, cuz we only want to use the custom proxy if USEPROXY is set, default target API without is crc ya'd do `API_ENDOINT=https://ci.cloud.redhat.com npm run start:proxy` to specify the proxy